### DISCUSSION TITLE
오늘 자정까지 pr을 못올릴 것 같습니다.

### DISCUSSION THREAD

**Discussion by hi-hyein** [ThinkAboutSoftware/AcademicConference#119] [General]
열심히 pr올리신 다른분들에게 죄송하게도..
오늘 자정까지는 pr을 못올릴 것 같아요
내일 자정까지 올릴 수 있도록 하겠습니다ㅠㅠ

**Comment by jongfeel** [comment]
그렇군요.

바쁘셔서 그런 거 같으니, 내일 잘 올려 주세요!
